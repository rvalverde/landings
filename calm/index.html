<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
	<title>Calm Template</title>
	<link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
	<link rel="stylesheet" href="https://cdn.raulvalverde.com/css/reset.css" type="text/css" media="screen" />
	<link rel="stylesheet" href="css/main.css" type="text/css" media="screen" />
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
	<script type="text/javascript" src="js/tinynav.min.js"></script>
	<script type="text/javascript">
		$(function () {
			$('#nav').tinyNav({
				active: 'selected',
				indent: 'â†’'
			});
		});
	</script>
</head>
<body>
	
	<div id="container">
		<header>
			<div class="width">
				<div class="block">
					<div class="logo">
						<a href="#"><img alt="" src="images/logo.png" /></a>
					</div>
					<nav class="menu">
						<ul id="nav">
							<li class="active"><a href="#">Home</a></li>
							<li><a href="#">About</a></li>
							<li><a href="#">Portfolio</a></li>
							<li><a href="#">Support</a></li>
							<li><a href="#">Contact</a></li>
						</ul>
					</nav>
				</div>
				<div class="block">
					<div class="welcome">
						<h1>Welcome Ladies & Gents to Calm Website.</h1>
						<p>A theme more stylish than any other on ThemeForest.</p>
					</div>
					<div class="find">
						<h2>Find Us Online</h2>
						<nav class="share">
							<ul>
								<li class="twiiter"><a href="#">Twitter</a></li>
								<li class="facebook"><a href="#">Facebook</a></li>
								<li class="pink"><a href="#">Pink</a></li>
								<li class="google"><a href="#">Google</a></li>
								<li class="green"><a href="#">Green</a></li>
							</ul>
						</nav>
					</div>
				</div>
			</div>
		</header>
		
		<section>
			<div class="width">
				<nav class="category">
					<ul class="filter-menu">
						<li><a href="#" data-filter="*" class="current">All</a></li>
						<li><a href="#" data-filter=".design">Design</a></li>
						<li><a href="#" data-filter=".branding">Branding</a></li>
						<li><a href="#" data-filter=".graphic">Graphic</a></li>
						<li><a href="#" data-filter=".animation">Animation</a></li>
						<li><a href="#" data-filter=".illustration">Illustration</a></li>
						<li><a href="#" data-filter=".photography">Photography</a></li>
					</ul>
				</nav>
				<div class="block filter-container">
					<article class="item design">
						<p><img alt="" src="images/section/001.jpg" /></p>
						<h2><a href="#">Working in Shop</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item branding">
						<p><img alt="" src="images/section/002.jpg" /></p>
						<h2><a href="#">Touch and Swipe</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item graphic">
						<p><img alt="" src="images/section/003.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item branding">
						<p><img alt="" src="images/section/004.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item illustration">
						<p><img alt="" src="images/section/005.jpg" /></p>
						<h2><a href="#">Working in Shop</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item photography">
						<p><img alt="" src="images/section/006.jpg" /></p>
						<h2><a href="#">Touch and Swipe</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item graphic">
						<p><img alt="" src="images/section/007.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item design">
						<p><img alt="" src="images/section/008.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item photography">
						<p><img alt="" src="images/section/009.jpg" /></p>
						<h2><a href="#">Working in Shop</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item animation">
						<p><img alt="" src="images/section/010.jpg" /></p>
						<h2><a href="#">Touch and Swipe</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item design">
						<p><img alt="" src="images/section/011.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item graphic">
						<p><img alt="" src="images/section/012.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item design">
						<p><img alt="" src="images/section/001.jpg" /></p>
						<h2><a href="#">Working in Shop</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item animation">
						<p><img alt="" src="images/section/002.jpg" /></p>
						<h2><a href="#">Touch and Swipe</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item graphic">
						<p><img alt="" src="images/section/003.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
					<article class="item branding">
						<p><img alt="" src="images/section/004.jpg" /></p>
						<h2><a href="#">What if you hire Arek</a></h2>
						<p><a href="#">Branding</a> / <a href="#">Design</a></p>
						<nav>
							<ul>
								<li class="zoom"><a href="#">Zoom</a></li>
								<li class="max"><a href="#">Max</a></li>
							</ul>
						</nav>
					</article>
				</div>
			</div>
		</section>
	</div><!-- @end #container -->
	
	<footer>
		<div class="width">
			<div class="block">
				<div class="item">
					<h3>About Us</h3>
					<p>Duis id eros ac eros cursus gravida a non magna. Sed vitae nisl diam, non sollicitudin mi. In dui arcu, rhoncus ac consequat eget, cursus sit amet risus.</p>
					<nav class="info">
						<ul>
							<li>Address: Street 9890, New Something 1234, Country</li>
							<li>Telephone: 1234 5678</li>
							<li>Fax: 9876 5432</li>
						</ul>
					</nav>
					<nav class="social">
						<ul>
							<li class="twiiter"><a href="#">Twitter</a></li>
							<li class="facebook"><a href="#">Facebook</a></li>
							<li class="instagram"><a href="#">Instagram</a></li>
							<li class="foursquare"><a href="#">Foursquare</a></li>
							<li class="flickr"><a href="#">Flickr</a></li>
						</ul>
					</nav>
				</div><!-- @end .item -->
				<div class="item">
					<h3>Latest Posts</h3>
					<nav class="posts">
						<ul>
							<li><a href="#">Curabitur pharetra, ante!</a> <span>25 April 2013</span></li>
							<li><a href="#">Gentle Marimba by Alastair Cameron</a> <span>01 May 2013</span></li>
							<li><a href="#">Curabitur pharetra, ante!</a> <span>25 April 2013</span></li>
							<li><a href="#">Gentle Marimba by Alastair Cameron</a> <span>01 May 2013</span></li>
						</ul>
					</nav>
				</div><!-- @end .item -->
				<div class="item">
					<h3>Latest Comments</h3>
					<nav class="comments">
						<ul>
							<li><a href="#">Dann Petty on Curabitur pharetra, ante lobortis dictum.</a></li>
							<li><a href="#">Kevin Kalde on Aliquam vulputate, pede vel vehicula accumsan, mi neque rutrum erat, eu congue orci lorem eget lorem. </a></li>
							<li><a href="#">Sean McCabe on Class aptent taciti sociosqu ad litora torquent per </a></li>
						</ul>
					</nav>
				</div><!-- @end .item -->
				<div class="item">
					<h3>Flickr Photostream</h3>
					<nav class="photos">
						<ul>
							<li><a href="#"><img alt="" src="images/photos/001.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/002.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/003.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/004.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/005.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/006.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/007.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/008.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/009.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/010.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/011.jpg" /></a></li>
							<li><a href="#"><img alt="" src="images/photos/012.jpg" /></a></li>
						</ul>
					</nav>
				</div><!-- @end .item -->
			</div>
			<div class="block footer">
				<nav class="options">
					<ul>
						<li><a href="#">Home</a></li>
						<li><a href="#">About</a></li>
						<li><a href="#">Portfolio</a></li>
						<li><a href="#">Blog</a></li>
						<li><a href="#">Features</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</nav>
				<p>Copyright &copy; Calm. All Rights Reserved</p>
			</div>
		</div>
	</footer>
		
</body>
</html>