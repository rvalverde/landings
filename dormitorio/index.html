<!DOCTYPE html>
<html lang="es">
<head>
	<meta charset="UTF-8" />
	<meta name="viewport" content="width=device-width, height=device-height, initial-scale=1, maximum-scale=1, user-scalable=no" >
	
	<title>Especial Invierno Dormitorio</title>
	
	<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet' type='text/css'>
	<link rel="stylesheet" type="text/css" href="css/style.css" />
</head>
<body>

	<div id="top">
		<a href="#container">Subir</a>
	</div>

	<div id="container">
		
		<section id="header">
			<div class="width">

				<div class="info">
					<div id="logo">
						<img alt="" src="images/logo.png" />
					</div>
				</div><!-- .info --> 

				<div class="slider">

					<div class="tag">
						<span>Colchones, Camas y Dormitorios</span>
						<span class="small">sueños reparadores</span>
					</div><!-- .tag --> 
					
					<div class="cycle-slideshow" data-cycle-swipe="true" data-cycle-slides="> div" data-cycle-timeout="3500" data-cycle-speed="1000" data-cycle-fx="fade" data-cycle-pager="#pager-1" data-cycle-pager-template="<span>{{slideNum}}</span>">
						<div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/1.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/2.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/3.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/4.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/5.jpg" />
							</a>
						</div><!-- .item -->
                        <div class="item">
							<a href="" target="_blank">
								<img alt="" src="images/slider/header/6.jpg" />
							</a>
						</div><!-- .item -->
					</div>
					<div id="pager-1" class="pager"></div>
				</div><!-- @end #slider -->
			</div>
		</section><!-- @end #slider -->
		
		<br class="clear" />
		
		<section id="home">
			<div class="width">
				<div id="products">
					<div class="group">
						<div class="item number-4 odd">
							<a href="" target="_blank">
								<img alt="" src="images/products/4.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item number-2 alt">
							<a href="" target="_blank">
								<img alt="" src="images/products/2.jpg" />
							</a>
						</div><!-- .item -->
					</div>
					<div class="group">
						<div class="item number-1 odd">
							<a href="" target="_blank">
								<img alt="" src="images/products/1.jpg" />
							</a>
						</div><!-- .item -->
						<div class="item number-3 alt">
							<a href="" target="_blank">
								<img alt="" src="images/products/3.jpg" />
							</a>
						</div><!-- .item -->
					</div>
				</div>
				
			</div>
		</section>
		
		<br class="clear" />
		
		<section id="bottom">
			<div class="slider">
				<div class="cycle-slideshow" data-cycle-slides="> div" data-cycle-timeout="3500" data-cycle-speed="1000" data-cycle-fx="fade" data-cycle-pager="#pager-2" data-cycle-pager-template="<span>{{slideNum}}</span>">
					<div class="item number-1">
						<img alt="" src="images/slider/footer/1.jpg" />
					</div><!-- .item -->
					<div class="item number-2">
						<img alt="" src="images/slider/footer/2.jpg" />
					</div><!-- .item -->
					<div class="item number-3">
						<img alt="" src="images/slider/footer/3.jpg" />
					</div><!-- .item -->
				</div>
				<div id="pager-2" class="pager"></div>
			</div><!-- @end #slider -->
			
			<div id="more">
				<a href="">Ver más productos</a>
			</div>
			
		</section>
		
	</div>

	<script type="text/javascript" src="js/jquery.min.js"></script>
	<script type="text/javascript" src="js/jquery.cycle2.min.js"></script>
	<script type="text/javascript" src="js/jquery.cycle2.swipe.min.js"></script>
	<script type="text/javascript" src="js/jquery.web.js"></script>
	
</body>
</html>