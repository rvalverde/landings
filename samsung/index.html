<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
	<title>Template</title>
	<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,800,600,700' rel='stylesheet' type='text/css'>
	<link rel="stylesheet" href="css/reset.css" type="text/css" media="screen" />
	<link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
	<!--[if gte IE 7]>
		<link rel="stylesheet" href="css/ie.css" type="text/css" media="screen" />
	<![endif]-->
	<script type="text/javascript" src="js/jquery.js" ></script>
	<script type="text/javascript" src="js/jquery.menutron.js"></script>
	<script type="text/javascript" src="js/jquery.main.js"></script>
</head>
<body>

	<div id="container">
	
		<div id="supertop">
			<div id="top">
				<div class="width">
					<div id="nav">
						<ul class="format">
							<li><a href="#" target="_blank">Category 1</a></li>
							<li><a href="#" target="_blank">Category 2</a></li>
							<li><a href="#" target="_blank">Category 3</a></li>
							<li><a href="#" target="_blank">Category 4</a></li>
							<li><a href="#" target="_blank">Category 5</a></li>
							<li><a href="#" target="_blank">Category 6</a></li>
							<li><a href="#" target="_blank">Category 7</a></li>
						</ul>
					</div>
				</div>
			</div>
			
			<div id="navigation">
				<ul>
					<li><a href="#colors">Customize it</a></li>
					<li><a href="#watch">Premiere</a></li>
					<li><a href="#notify">Always Connected</a></li>
					<li><a href="#featured">More Features</a></li>
				</ul>
			</div>
		</div>
		
		<div id="samsung">
		
			<div id="gear" class="page">
				<div class="width">
					<div class="subpage">
						<div class="content">
							<h1><img alt="" src="images/samsung.png" /></h1>
							<ul>
								<li>Lift a Samsung Gear Fit</li>
								<li><strong>+ 3 Straps FREE</strong></li>
								<li><strong>+ 1 Charging Pod</strong></li>
							</ul>
							<span class="normal">Normal price: $ 899</span>
							<span class="price">Only <span>$399</span></span>
							<span class="buy"><a href="">Buy Now</a></span>
						</div>
						<div class="image">
							<img alt="" src="images/products/gear.jpg" />
						</div>
					</div>
				</div>
			</div><!-- # -->
			
			<div id="colors" class="page">
				<div class="width">
					<div class="subpage">
						<div class="content">
							<h2>Interchangeable belts</h2>
							<p>Samsung Gear Fit's advanced curved Super AMOLED touch screen and interchangeable bands* make you look stylish in all your activities.</p>
							<p><span class="buy small"><a href="">Buy Now</a></span></p>
						</div>
						<div class="circles">
							<h3>Available Colors</h3>
							<img alt="" src="images/circles.png" />
						</div>
						<div class="image">
							<img alt="" src="images/products/colors.jpg" />
						</div>
					</div>
				</div>
			</div><!-- # -->
			
			<div id="watch" class="page">
				<div class="width">
					<div class="subpage">
						<div class="content">
							<h2>Your personal trainer</h2>
							<p>Through its heart rate sensor, Samsung Gear Fit gives you real-time information on the progress and results of your sports activity, as well as personalized recommendations to help you exercise according to your needs.</p>
							<p><span class="buy small"><a href="">Buy Now</a></span></p>
						</div>
						<div class="image">
							<img alt="" src="images/products/watch.jpg" />
						</div>
					</div>
				</div>
			</div><!-- # -->
			
			<div id="notify" class="page">
				<div class="width">
					<div class="subpage">
						<div class="content">
							<h2>Interchangeable belts</h2>
							<p>Samsung Gear Fit's advanced curved Super AMOLED touch screen and interchangeable bands* make you look stylish in all your activities.</p>
							<p><span class="buy small"><a href="">Buy Now</a></span></p>
						</div>
						<div class="image">
							<img alt="" src="images/products/notify.jpg" />
						</div>
					</div>
				</div>
			</div><!-- # -->
			
			<div id="featured">
				<div class="item">
					<div class="width">
						<div id="featured-1" class="float-left">
							<div class="block">
								<div class="content">
									<h3>Customize your Gear Fit</h3>
									<p>You can customize your Samsung Gear Fit with a wide range of colors, different wallpapers, and various types of watches to set it up just the way you like it.</p>
								</div>
								<div class="thumb">
									<img alt="" src="images/products/featured/1.jpg" />
								</div>
							</div>
						</div>
						<div id="featured-2" class="float-right">
							<div class="block">
								<div class="content">
									<h3>Always wear it</h3>
									<p>Like you, Samsung Gear Fit never stops. It's ready to continuously monitor and record your activities throughout the day. Whatever adventure you're on, Gear Fit is ready, thanks to its water and dust resistance.</p>
								</div>
								<div class="thumb">
									<img alt="" src="images/products/featured/2.jpg" />
								</div>
							</div>
						</div>
					</div>
				</div><!-- .item -->
				<div class="item alt">
					<div class="width">
						<div id="featured-3" class="float-left">
							<div class="block">
								<div class="content">
									<h3>Compatibility</h3>
									<p>Gear Fit is compatible with multiple Galaxy family smartphones and tablets.</p>
									<p>Samsung Gear is compatible with 17 mobile devices. Support for other models will be increased soon.</p>
								</div>
								<div class="thumb">
									<img alt="" src="images/products/featured/4.jpg" />
								</div>
							</div>
						</div>
						<div id="featured-4" class="float-right">
							<div class="block">
								<div class="content">
									<h3>Simple Interaction</h3>
									<p>With Gear Fit you can control basic tasks on your smartphone, such as rejecting calls by sending a message, quick replies, or setting alarms.</p>
								</div>
								<div class="thumb">
									<img alt="" src="images/products/featured/3.jpg" />
								</div>
							</div>
						</div>
					</div>
				</div><!-- .item -->
			</div><!-- # -->
			
		</div><!-- #samsung -->
		
		<div id="footer">
			<div class="width">
        Copyright &copy; All right reserved.
      </div>
		</div>
	</div>
</body>
</html>