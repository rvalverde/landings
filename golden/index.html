<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
	<title>Golden Template</title>
	<link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css' />
	<link href='https://fonts.googleapis.com/css?family=Droid+Serif:400italic' rel='stylesheet' type='text/css'>
	<link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,700' rel='stylesheet' type='text/css'>
	<link rel="stylesheet" href="https://cdn.raulvalverde.com/css/reset.css" type="text/css" media="screen" />
	<link rel="stylesheet" href="css/main.css" type="text/css" media="screen" />
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
	<script type="text/javascript" src="js/tinynav.min.js"></script>
	<script type="text/javascript">
		$(function () {
			$('#nav').tinyNav({
				active: 'selected',
				indent: '→'
			});
		});
	</script>
</head>
<body>
	
	<div id="container">
		<header>
			<div class="width">
				<div class="block">
					<div class="logo">
						<a href="#"><img alt="" src="images/logo.png" /></a>
					</div>
					<nav class="menu">
						<ul id="nav">
							<li class="active"><a href="#">Home</a></li>
							<li><a href="#">Services</a></li>
							<li><a href="#">Portfolio</a></li>
							<li><a href="#">About</a></li>
							<li><a href="#">Contact</a></li>
						</ul>
					</nav>
					<h1>Welcome To Our Studio!s</h1>
					<h2>It’s nice to Meet you</h2>
					<p class="text-center"><a href="#" class="button-submit">Tell Me More</a></p>
				</div>
			</div>
		</header>
		
		<section>
			<article id="services">
				<div class="width">
					<div class="title">
						<h2>Services</h2>
						<p>Proin iaculis purus consequat sem cure.</p>
					</div><!-- @end .title -->
					<div class="block">
						<div class="item">
							<h3>E-Commerce</h3>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
						</div>
						<div class="item">
							<h3>Responsive Web</h3>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
						</div>
						<div class="item">
							<h3>Web Security</h3>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
						</div>
					</div>
				</div>
			</article>
			<article id="portfolio">
				<div class="width">
					<div class="title">
						<h2>Our Portfolio</h2>
						<p>Proin iaculis purus consequat sem cure.</p>
					</div><!-- @end .title -->
					<div class="block">
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/burned.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Ebony & Ivory</a></h3>
								<p><a href="#">Brading</a></p>
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/smart.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Smart Stationary</a></h3>
								<p><a href="#">Print Design</a></p>
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/clever.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Clever Poster</a></h3>
								<p><a href="#">Print Design</a></p>
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/vinyl.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Vinyl Record</a></h3>
								<p><a href="#">Product Mock-Up</a></p>
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/treehouse.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Treehouse Template</a></h3>
								<p><a href="#">Web Design</a></p>
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb">
								<img alt="" src="images/portfolio/burned.jpg" />
							</div>
							<div class="info">
								<h3><a href="#">Burned Logo</a></h3>
								<p><a href="#">Brading</a></p>
							</div>
						</div><!-- @end .item -->
					</div>
				</div>
			</article>
			<article id="about">
				<div class="width">
					<div class="title">
						<h2>About Us</h2>
						<p>Proin iaculis purus consequat sem cure.</p>
					</div><!-- @end .title -->
					<div class="block">
						<div class="item">
							<div class="left">
								<h3>July 2010</h3>
								<h4>Our Humble Beginnings</h4>
								<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="center">
								<img alt="" src="images/about/001.jpg" />
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="left">
								<h3>January 2011</h3>
								<h4>Facing Startup Battles</h4>
								<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="center">
								<img alt="" src="images/about/002.jpg" />
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="left">
								<h3>December 2012</h3>
								<h4>Enter The Dark Days</h4>
								<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="center">
								<img alt="" src="images/about/003.jpg" />
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="left">
							<h3>February 2014</h3>
								<h4>Our Triumph</h4>
								<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="center">
								<img alt="" src="images/about/004.jpg" />
							</div>
						</div><!-- @end .item -->
						<div class="item">
							<div class="left"></div>
							<div class="center">
								<a href="#" class="more"><span>Our</span> <span>Story</span> <span>Continues</span> <span>...</span></a>
							</div>
						</div><!-- @end .item -->
					</div>
				</div>
			</article>
			<article id="team">
				<div class="width">
					<div class="title">
						<h2>Our Amazing Team</h2>
						<p>Proin iaculis purus consequat sem cure.</p>
					</div><!-- @end .title -->
					<div class="block">
						<div class="item">
							<div class="thumb"></div>
							<h3><a href="#">Kimberly Thompson</a></h3>
							<p><a href="#">Marketer</a></p>
							<ul>
								<li><a href="#"><span class="icon twitter"></span></a></li>
								<li><a href="#"><span class="icon facebook"></span></a></li>
								<li><a href="#"><span class="icon pinterest"></span></a></li>
							</ul>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb"></div>
							<h3><a href="#">Rico Massino</a></h3>
							<p><a href="#">Coder</a></p>
							<ul>
								<li><a href="#"><span class="icon twitter"></span></a></li>
								<li><a href="#"><span class="icon facebook"></span></a></li>
								<li><a href="#"><span class="icon pinterest"></span></a></li>
							</ul>
						</div><!-- @end .item -->
						<div class="item">
							<div class="thumb"></div>
							<h3><a href="#">Uku Mason</a></h3>
							<p><a href="#">Graphic Designer</a></p>
							<ul>
								<li><a href="#"><span class="icon twitter"></span></a></li>
								<li><a href="#"><span class="icon facebook"></span></a></li>
								<li><a href="#"><span class="icon pinterest"></span></a></li>
							</ul>
						</div><!-- @end .item -->
					</div>
					<div class="info">
						<p class="text-center">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
					</div>
				</div>
			</article>
			<article id="partners">
				<div class="width">
					<img alt="" src="images/partners.jpg" class="aligncenter" />
				</div>
			</article>
		</section>
	</div><!-- @end #container -->
	
	<footer>
		<div id="footer">
			<div class="width">
				<div class="title">
					<h2>Contact Us</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
					<div class="form-block form-horz">
						<div class="block">
							<div class="float-left">
								<div class="form-group">
									<input type="text" placeholder="your Name *" />
								</div>
								<div class="form-group">
									<input type="text" placeholder="your e-mail *" />
								</div>
								<div class="form-group">
									<input type="text" placeholder="subject" />
								</div>
							</div>
							<div class="float-right">
								<div class="form-group">
									<textarea placeholder="Your message *"></textarea>
								</div>
							</div>
						</div>
						<div class="block">
							<div class="form-group">
								<input type="submit" value="Send Message" class="button-submit" />
							</div>
						</div>
					</div>
				</div><!-- @end .title -->
			</div>
		</div>
		<div id="bottom">
			<div class="width">
				<ul>
					<li><a href="#"><span class="icon black twitter"></span></a></li>
					<li><a href="#"><span class="icon black facebook"></span></a></li>
					<li><a href="#"><span class="icon black pinterest"></span></a></li>
					<li><a href="#"><span class="icon black google"></span></a></li>
				</ul>
				<p>Copyright &copy; All right reserved.</p>
			</div>
		</div>
	</footer>
		
</body>
</html>